src/main/java/org/example/constant/TableContent.java

```diff
@@ -0,0 +1,20 @@
+package org.example.constant;
+
+public enum TableContent {
```

Comment:
Move all variables into Developer model.

They should be constants.
Then on the JSP, you can grab them.

Reply:
fixed